src/pudl/analysis/plant_parts_eia.py

```diff
@@ -147,33 +147,13 @@
 
 **Generating the plant-parts list**
 
-There are two ways to generate the plant-parts table: one directly using the
-:class:`pudl.output.pudltabl.PudlTabl` object and the other using the classes
-from this module. Either option needs a :class:`pudl.output.pudltabl.PudlTabl`
-object.
-
-Create the :class:`pudl.output.pudltabl.PudlTabl` object:
-
-.. code-block:: python
-
-    import pudl
-    from pudl.workspace.setup import PudlPaths
-    pudl_engine = sa.create_engine(PudlPaths().pudl_db)
-    pudl_out = pudl.output.pudltabl.PudlTabl(pudl_engine,freq='YS')
-
-Then make the table via pudl_out:
-
-.. code-block:: python
-
-    plant_parts_eia = pudl_out.plant_parts_eia()
-
-
-OR make the table via objects in this module:
```

Comment:
Removed because this is, I think, no longer accurate given that `PudlTabl` is just a wrapper around the PUDL DB now.